### ISSUE TITLE
No class present to indicate state of dropdown

### ISSUE THREAD

**Issue by ismay** [RSSchermer/ember-rl-dropdown#14]
It would be helpful to have an `is-expanded` class present on the dropdown-container when it's expanded. That way you could define styles for when the dropdown is expanded (for example, updating a toggle icon).

Same thing for the template:
```
{{#rl-dropdown-toggle-expanded}}
  {{! expanded icon }}
{{/rl-dropdown-toggle-expanded}}
{{#rl-dropdown-toggle-collapsed}}
  {{! collapsed icon }}
{{/rl-dropdown-toggle-collapsed}}
```
Or something similar.

**Comment by RSSchermer** [comment]
Thanks for the suggestion of adding a class indicated dropdown expansion to the container component. As of 0.6.0 the container component should have the `dropdown-expanded` class when the dropdown is expanded.

I'm not entirely sure what you mean with the template example, could you elaborate?

**Comment by ismay** [comment]
Thanks for the update! What I meant with the template example is the following: let's say you want to display a `+` icon when the dropdown is closed, and a `-` when it's open. It's currently impossible to do that in the templates with ember-rl-dropdown. It would be nice if this would be possible:

```handlebars
{{#rl-dropdown-container}}
  {{#rl-dropdown-toggle-expanded}}
    - Click to close
  {{/rl-dropdown-toggle-expanded}}
  {{#rl-dropdown-toggle-collapsed}}
    + Click to open
  {{/rl-dropdown-toggle-collapsed}}
  {{#rl-dropdown}}
    Dropdown here
  {{/rl-dropdown}}
{{/rl-dropdown-container}}

```

Depending on the state it would show one or the other.

**Comment by RSSchermer** [comment]
Ah, I see. Yea, I agree that it'd be nice if the toggle button text could be different based on whether the dropdown is expanded. Perhaps block params would be a nice way of accomplishing that:

```hbs
{{#rl-dropdown-container as |isExpanded|}}
  {{#rl-dropdown-toggle}}
    {{#if isExpanded}}
      - Click to close
    {{else}}
      + Click to open
    {{/if}}
  {{/rl-dropdown-toggle}}

  {{#rl-dropdown}}
    Dropdown here
  {{/rl-dropdown}}
{{/rl-dropdown-container}}
```

I'll look into it.

**Comment by ismay** [comment]
Yeah that looks like a nice way to do it. Better than what I suggested.

**Comment by RSSchermer** [comment]
I just released 0.7.0, which implements what I described above. Hope that gives you everything you need for your use-case!

**Comment by ismay** [comment]
@RSSchermer That's perfect, works like a charm. Thanks!
